app/js/controllers/sellCtrl.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by student on 06.12.16.
+ */
+(function ()
+{
+    'use strict';
+    angular.module('cinkciarzTraining')
+            .controller('SellController', SellController);
+    function SellController($scope, $routeParams, WalletService, CurrenciesService, $timeout, $window)
+    {
+        var vm = this;
+        vm.currency = $routeParams.currency;
+        vm.rates = {};
+        vm.showTitle = showTitle;
+        vm.sell = sell;
+        vm.divShow = false;
+        vm.wallet = WalletService.getWallet();
+        vm.validateValue = validateValue;
+        vm.message = '';
+        vm.back = back;
+        getCurrencies();
+
+
+        ///////////////////////
+        function showTitle()
+        {
+            if ($routeParams.currency === 'EUR') {
+                return 'Euro';
+            } else if ($routeParams.currency === 'USD') {
+                return 'Dolarów';
+            } else if ($routeParams.currency === 'GBP') {
+                return 'Funtów';
+            }
+        }
+
+        function sell()
+        {
+            if (validateValue()) {
+                return;
```

Comment:
something like that you can use without else, return do the work and it will be less code